recipe/meta.yaml

```diff
@@ -66,5 +66,6 @@ about:
 
 extra:
   recipe-maintainers:
+    - Guthur
     - osallou
     - bollwyvl
```

Comment:
```suggestion
```
@bollwyvl did you want to remove yourself. sorry again for adding you in accident.

Reply:
No, it's fine, happy to help, and might pick up some of the older downstreams. Better to keep the scope of this simpler.

Reply:
ok. thank you and apologies again.